docs/integration-guides/expo-react-native.md

```diff
@@ -93,21 +97,22 @@ import AddAppAdminConsole  from '../includes/_add-application-console.mdx';
 
   | Property | Value | 
   | ----------- | ----------- |
-  | **Protocol** | <mark>What's recommended for this particular use case?</mark> |
+  | **Protocol** | OIDC<br /><ProtocolOIDC/> |
   | **Client Type** | <mark>What's recommended for this particular use case?</mark> | 
-  | **PKCE** | <mark>What's recommended for this particular use case?</mark> | 
-  | **Redirect URIs** | Use your application's App Scheme or Universal URL.<br /><br />Your real redirect URI follows the pattern:<br /><br /> <mark>Can we provide an example?</mark> | 
-  | **Token Endpoint Auth Method** | <mark>What's recommended for this particular use case?</mark> | 
-  | **Grant Type** | <mark>What's recommended for this particular use case?</mark> | 
+  | **PKCE** | S256<br /><PKCES256/> |
+  | **Redirect URIs** | Use your application's App Scheme or Universal URL.<br /><br />If you are using an app scheme, your redirect URI may follow the pattern:<br /><br /> `myapp://` <br /> <br />Follow Expo's [deep linking guide](https://docs.expo.dev/guides/deep-linking/) and [linking to your development build](https://docs.expo.dev/guides/linking/#linking-to-your-app) | 
+  | **Token Endpoint Auth Method** | Client Secret Post | 
+  | **Grant Type** | Authorization Code<br /><GrantTypeAuthorizationCode/> |
   | **All other options** | Use the default values for the remaining options |  
 
-1. Click the **Authenticator Config** tab, select **Embedded SDK** as the Configuration Type, and use the following values. 
+1. Click the **Authenticator Config** tab and use the following values. 
 
   | Property | Value | 
   | ----------- | ----------- |
-  | **Invocation Type** | <mark>What's recommended for this particular use case?</mark> |
+  | **Configuration Type** | Embedded SDK |
```

Comment:
nit: We added "Why Hosted Web?". Can we add "Why Embedded?" as well? 🙂

Reply:
Yes, but maybe in a follow up!